src/review-form.ts

```diff
@@ -16,11 +16,11 @@ import type {
   RecaptchaWidgetInterface,
 } from '@internetarchive/recaptcha-manager';
 import '@internetarchive/ia-activity-indicator';
+import { Review } from '@internetarchive/metadata-service';
```

Comment:
Minor, but this can be `import type { Review } from ...` since you're not instantiating a `Review` in here, only referencing the type.